test/stress/CMakeLists.txt

```diff
@@ -0,0 +1,122 @@
+set (CVMFS_STRESS_SOURCES
```

Comment:
That looks a bit much, are you sure we need all of these source files?  E.g. do we need the swissknife... components?